app/jobs/shipit/refresh_check_runs_job.rb

```diff
@@ -0,0 +1,14 @@
+module Shipit
+  class RefreshCheckRunsJob < BackgroundJob
+    queue_as :default
+
+    def perform(params)
+      if params[:commit_id]
+        Commit.find(params[:commit_id]).refresh_check_runs!
+      else
+        stack = Stack.find(params[:stack_id])
+        stack.commits.order(id: :desc).limit(30).each(&:refresh_check_runs!)
```

Comment:
It's mostly arbitrary (and copied over from `RefreshStatusesJob`. When you create a new stack Shipit will fetch up to 30 commits, so that number is reused here. I should probably DRY it up.